### DISCUSSION TITLE
Realtime Connection to Github Code

### DISCUSSION THREAD

**Discussion by jayonsoftware** [GitbookIO/community#189] [Feature Requests]
### What's your feature/idea?

Currently every time I update the code examples in Github, I have to cut and paste that code into Gitbook. Can we have a function where we can just directly link and Gitbook will pull the latest. That will save me a lot of time and also keep things in sync

### What's your desired outcome?

Use it :)

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by pr0f37** [comment]
How about integrating gitbook with your github repository's code examples folder? it's explained here: https://www.youtube.com/watch?v=Fm5hYBsRSXo

**Reply by jayonsoftware** [reply]
I look at the video, but may be I am missing some thing here. Here is what I am doing now.

Whenever a sample code gets updated, I have to go to git, cut that code, and paste it into a Codeblock on Gitbook. It would be nice if I could just post the link to the code in the code block and the code block shows the latest. I can use the link to Git Gist function, but then again I have to copy the sample code from the repo to a gist.
